chain/client-primitives/src/types.rs

```diff
@@ -1049,6 +1049,13 @@ impl From<std::io::Error> for GetSplitStorageInfoError {
     }
 }
 
+#[derive(Debug)]
+pub struct GetTailBlockHeight;
```

Comment:
Isn't there an existing endpoint to query for tail? I'm not saying there is, just checking.